### PR TITLE
Update how-to-extend-otrs.rst

### PR DESCRIPTION
**Pull request by MIPMHannes**
This example is broken because `1;` is ignored

### CODE REVIEW

File: source/developer/contributing/how-to-extend-otrs.rst [hunk 1]
```diff
@@ -259,10 +259,9 @@ Next, we create the file for the core module ``/HelloWorld/Kernel/System/HelloWo
         return $HelloWorld;
 
     }
-
+    1;
     =end Internal:
```
Comment (dennykorsukewitz) [inline source/developer/contributing/how-to-extend-otrs.rst]:
```suggestion
   
    =end Internal:

    =cut
    
     1;
```

**Review by dennykorsukewitz — CHANGES_REQUESTED** [review]
Hi  @MIPMHannes ,
thanks for your PR.

I have a few change requests here.

`=cut` is missing...

Regards :rocket:

**Comment by dennykorsukewitz** [comment]
Hi @MIPMHannes 

can you please verify the changes.

Thanks.

**Review by MIPMHannes — COMMENTED** [review]
You are right. I think this is the correct way

**Review by rkaldung — APPROVED** [review]

**Pull request merged**